### PR TITLE
Testing @seb32 ping

### PR DESCRIPTION
**Pull request by Sebastien32**
hi endrawes

### CODE REVIEW

**Comment by endrawes0** [comment]
Please add a summary and description

**Comment by endrawes0** [comment]
Also, this breaks our lint rules. I'll explain inline.

File: src/watcher.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ function Watcher(bot, league) {
             self.bot.say({
                 text: "<@" + white + ">,  <@" + black + ">:"
                     + " Your game is *not valid* because "
-                    + "*" + result.reason + "*",
+                    + "*" + result.reason + "*" + "@seb32",
```
Comment (endrawes0) [inline src/watcher.js]: The linter is complaining about this line.

It was you to remove the unnecessary concatenation.
 
`"...*"+"@seb32"`

Replace with a single string constant.

Comment (endrawes0) [inline src/watcher.js]: Also, this won't work unless you have notifications on for strings matching your name.

You need to use the user ID and Mark up to identify it as a special token.

Comment (Sebastien32) [inline src/watcher.js]: ok, will check how the other ones are done

Comment (endrawes0) [inline src/watcher.js]: 👍

**Pull request closed without merging**